src/index.js

```diff
@@ -1,29 +1,56 @@
 import './style.css';
-import { add, display } from './crud.js';
+import { createGame, sendData, getData } from './crud.js';
 
 const name = document.getElementById('name');
 const score = document.getElementById('score');
 
 const subBtn = document.querySelector('.subBtn');
 const refreshBtn = document.querySelector('.refreshBtn');
+const scoreSec = document.querySelector('.listScores');
 
-// setting up array of scores
-let scores = [];
+const baseURL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/';
 
-// setting up localstorage
-if (JSON.parse(localStorage.getItem('scores') !== null)) {
-  scores = JSON.parse(localStorage.getItem('scores'));
-} else {
-  localStorage.setItem('scores', JSON.stringify(scores));
-}
+const getIdFromStorage = () => {
+  const localStorageId = localStorage.getItem('id') ? JSON.parse(localStorage.getItem('id')) : null;
+  return localStorageId;
+};
+
+const display = async () => {
+  scoreSec.innerHTML = '';
+  const { result } = await getData(`${baseURL}games/${getIdFromStorage()}/scores`);
+  result.forEach((itemScore) => {
+    const eachScore = `<p class="eachScore">${itemScore.user}:${itemScore.score}</p><hr>`;
+    scoreSec.insertAdjacentHTML('beforeend', eachScore);
+  });
+};
+
+const saveGameOnLocalStorage = () => {
+  const data = {
+    name: 'Game of Legends',
+  };
+  if (!getIdFromStorage()) {
+    window.addEventListener('load', async () => {
+      const { result } = await createGame(`${baseURL}games`, data);
+      localStorage.setItem('id', JSON.stringify(result.substr(14, 20)));
+    });
+  }
+};
+
+const add = async (name, score) => {
+  const data = {
+    user: name,
+    score,
+  };
+
+  await sendData(`${baseURL}games/${getIdFromStorage()}/scores`, data);
+  display();
+};
 
 subBtn.addEventListener('click', () => {
-  add(name.value, score.value, scores);
-  localStorage.setItem('scores', JSON.stringify(scores));
-  display(scores);
+  add(name.value, score.value);
 });
 
 refreshBtn.addEventListener('click', () => {
-  localStorage.clear();
-  display(score);
-});
\ No newline at end of file
+  window.location.reload();
+});
+saveGameOnLocalStorage();
```

Comment:
- [ ]  Great job so far :+1:  , however, when you click the refresh button it does not reach the GET API endpoint, please make sure that the refresh button work as expected to reflect new information on the score board.